libs/s25main/Loader.cpp

```diff
@@ -343,6 +343,38 @@ void Loader::LoadDummyGUIFiles()
     }
 }
 
+namespace {
+struct NationResourcesSource
+{
+    bfs::path buildingsFilePath, iconsFilePath;
+};
+
+NationResourcesSource getNationResourcesSource(const Nation nation, bool isWinter, const RttrConfig& config)
+{
+    const auto shortName = std::string(NationNames[nation], 0, 3);
+    auto buildingsFilename = shortName + "_Z.LST";
+    auto iconsFilename = shortName + "_ICON.LST";
+    if(isWinter)
+        buildingsFilename.insert(buildingsFilename.begin(), 'W');
+    bfs::path nationFolder;
+    if(rttr::enum_cast(nation) < NUM_NATIVE_NATIONS)
+    {
+        nationFolder = config.ExpandPath(s25::folders::mbob);
```

Comment:
perhaps some note (=comment) here why we do this differentation

Reply:
What would you write here?
IMO the code is clean enough: For native nations use 1 folder and uppercase names and for non-native use another folder and lowercase names.
Can you make a suggestion (i.e. from a readers perspective)?

Reply:
`For native nations we use uppercase names and for non-native ("new" ones) we use lowercase names.`